src/sections/SectionHeading/notification.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './styles.css';
+
+export default function Notification(props) {
+    const {
+        notificationTitle,
+        notificationContent,
```

Comment:
title and content, the component already is named Notification it is too redundant